### PR TITLE
Added doc strings and  refactored code

### PR DESCRIPTION
**Pull request by isha1610200**

### CODE REVIEW

File: ase/search/models.py [hunk 1]
```diff
@@ -3,6 +3,13 @@
 
 # Create your models here.
 class Search(models.Model):
+    """
+    -- Summary --
```
Comment (NethraGunti) [inline ase/search/models.py:7]: Remove this line please

File: ase/search/models.py [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
+    Search model has query details.
+    #Field Description
+    > query is a character filed with max possible length = 100, which is basically the user query.
```
Comment (NethraGunti) [inline ase/search/models.py:10]: Try not to use general English and avoid specifying lengths/defaults
```suggestion
    > query is a character filed which keeps track of all the queries made onsite
```

File: ase/search/views.py [hunk 3]
```diff
@@ -14,6 +14,10 @@ def home(request):
 
 @login_required(redirect_field_name='next', login_url=reverse_lazy('login'))
 def query(request, argument):
+    """
+    -- Summary --
```
Comment (NethraGunti) [inline ase/search/views.py:18]: Remove this line

File: ase/users/models.py [hunk 4]
```diff
@@ -7,6 +7,15 @@
 # Create your models here.
 
 class user(AbstractUser):
+    """
+    -- Summary --
```
Comment (NethraGunti) [inline ase/users/models.py:11]: Remove this line please

File: ase/users/models.py [hunk 5]
```diff
@@ -7,6 +7,15 @@
...
+    User model has the details of the user.
+    #Field Descriptions
+    > username is a character field with maximum possible length = 25 which is basically the username of the user and is unique for every user.
+    > email is the email id of the user
```
Comment (NethraGunti) [inline ase/users/models.py:15]: Refer to this comment for changes
https://github.com/Sayam753/ASEProject/pull/1/files#r456696110

File: ase/users/models.py [hunk 6]
```diff
@@ -7,6 +7,15 @@
...
+    > phone_no is a character field, which is basically the phone number of the user
+
```
Comment (NethraGunti) [inline ase/users/models.py:17]: Refer to this comment for changes
https://github.com/Sayam753/ASEProject/pull/1/files#r456696110

File: ase/users/views.py [hunk 7]
```diff
@@ -5,6 +5,11 @@
 
 # Create your views here.
 def register(request):
+    """
+    -- Summary --
```
Comment (NethraGunti) [inline ase/users/views.py:9]: Remove this line please

File: ase/search/models.py (hunk 1 above)
Comment (isha1610200) [reply ase/search/models.py]: I have removed the same.

File: ase/search/models.py (hunk 2 above)
Comment (isha1610200) [reply ase/search/models.py]: Done

File: ase/search/views.py (hunk 3 above)
Comment (isha1610200) [reply ase/search/views.py]: Done

File: ase/users/models.py (hunk 4 above)
Comment (isha1610200) [reply ase/users/models.py]: Done

File: ase/users/models.py (hunk 6 above)
Comment (isha1610200) [reply ase/users/models.py]: Changed

File: ase/users/views.py (hunk 7 above)
Comment (isha1610200) [reply ase/users/views.py]: Done

**Comment by NethraGunti** [comment]
1. Added doc strings for existing code
2. Refactored existing code
3. Applied pep8 conventions for tests

**Pull request merged**